CSS/style.css

```diff
@@ -0,0 +1,903 @@
+body {
+  background-color: #f4f5f7;
+  width: 100vw;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  width: 100vw;
+  position: relative;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  background-color: white;
+}
+
+#logo {
+  text-decoration: none;
+  color: #6070ff;
+  margin: 24px auto 24px 20px;
+  width: 100vm;
+  height: 20px;
+  font-size: 18px;
+  display: flex;
+  font-weight: 700;
+}
+
+p,
+a {
+  font-size: larger;
+}
+
+h1,
+h2 {
+  font-size: x-large;
+}
+
+.nav-pic {
+  height: 12px;
+  padding-right: 15px;
+}
+
+.nav-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 15px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #000;
+}
+
+form li {
+  margin-bottom: 20px;
+}
+
+.nav-ul li {
+  display: flex;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  color: #344563;
+  margin-left: 32px;
+}
+
+.nav-ul a {
+  text-decoration: none;
+  transition: transform 0.2s;
+}
+
+.home {
+  background-image: url("../img/background.png");
+  background-color: white;
+  background-size: cover;
+  border-bottom-left-radius: 70px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: 50px 24px;
+  padding-right: 0;
+}
+
+.home h1 {
+  font-size: 40px;
+  margin-bottom: 0;
+  font-weight: 900;
+  line-height: 52px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.home p {
+  width: 270px;
+  font-size: 16px;
+}
+
+.home-content {
+  /* margin: 0% 0% 0% 8%; */
```

Comment:
- [ ] kindly as suggested by the previous reviewer, it would be better to remove all commented lines in your codes to keep it clean and professional 🙏